kong/plugins/kubernetes-sidecar-injector/api.lua

```diff
@@ -37,6 +37,23 @@ local function skip_injection(plugin_config, review_request) -- luacheck: ignore
 end
 
 
+-- Removes unknown fields from the request (i.e. from more recent Kubernetes versions)
+local function remove_unknown_keys(schema, input)
+  local result = {}
+  for k, field in schema:each_field() do
```

Comment:
I think we should only enter the `for` loop if `type(input) == "table"`.

Reply:
Done, added an `if` before `local result = {}` which early-exits